### PR TITLE
Don't load the bibliography file until main()

### PR DESCRIPTION
**Pull request by eric-wieser**
This might help a little with running this in third-party projects

### CODE REVIEW

**Comment by eric-wieser** [comment]
#deploy

File: print_docs.py [hunk 1]
```diff
@@ -81,69 +81,70 @@
...
-    journal = data.fields['booktitle']
-  else:
-    journal = None
-  data.fields['url'] = url
-  data.fields['journal'] = journal
-  data.backrefs = []
+      journal = None
+    data.fields['url'] = url
+    data.fields['journal'] = journal
+    data.backrefs = []
```
Comment (eric-wieser) [inline print_docs.py:147]:
```suggestion
    data.backrefs = []

  return bib
```

**Comment by eric-wieser** [comment]
#deploy

**Comment by bryangingechen** [comment]
LGTM. Thanks!

**Pull request merged**